Gems/LmbrCentral/Code/Source/Shape/EditorAxisAlignedBoxShapeComponent.cpp

```diff
@@ -171,4 +171,13 @@ namespace LmbrCentral
     {
         return AZ::Vector3(m_aaboxShape.GetCurrentTransform().GetUniformScale());
     }
+
+    AZ::Aabb EditorAxisAlignedBoxShapeComponent::GetLocalBounds()
```

Comment:
question: Should be implement `GetWorldBounds` too? (or is there a default implementation implemented in terms of this?)

Reply:
EditorBaseShapeComponent (which this component inherits from) already implements both GetLocalBounds and GetWorldBounds. Because of the unique way the AA box works, the base implementation of GetLocalBounds does not work for it, but the base implementation of GetWorldBounds does.

Reply:
Aaah I see perfect, makes sense thanks 👍